LocalSettings.php

```diff
@@ -3042,6 +3042,20 @@
 	'wgWhitelistRead' => [
 		'default' => [],
 	],
+	'wgDisabledVariants' => [
+		'default' => [],
+		'hkrailwiki' => [
+			'zh',
+			'zh-hant',
+			'zh-hans',
+		],
+	],
+	'wgDefaultLanguageVariant' => [
+		'default' => [],
+		'hkrailwiki' => [
+			'zh-hk',
+		],
+	],
```

Comment:
```suggestion
	'wgDefaultLanguageVariant' => [
		'default' => false,
		'hkrailwiki' => 'zh-hk',
	],
```

Reply:
```suggestion
	'wgDefaultLanguageVariant' => [
		'default' => false,
		'hkrailwiki' => 'zh-hk',
	],
```
@ugochimobi It would be like this. cc @Universal-Omega

Reply:
I just went ahead and applied the change myself. I'll merge this now.